src/components/modern-question-card/modern-question-card.tsx

```
@@ -425,7 +456,7 @@ const QuestionContent = ({
         )}
       </div>
       {topic && (
-        <div className="mt-4 flex flex-row gap-2 justify-between items-center">
+        <div className="mt-4 flex flex-row gap-2 justify-between items-center cursor-pointer" onClick={onClickTopic}>
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)